README.md

```diff
@@ -1,48 +1,207 @@
 # ActiveRecord Validations
 
-## Objectives
+ActiveRecord can validate our models for us before they even touch the
+database.
 
-Students will be able to:
+We can use `ActiveRecord::Base` class methods like `#validates()` to set things
+up.
+
+
+# Objectives
+
+After this lesson, you should be able to:
 
-- Apply a validation requirement using validates macro
 - Identify when validation occurs in the lifespan of an object
-- Introspect on the ActiveRecord::Errors collection object on an AR instance
-  - use valid?
-  - use errors
-- Generate full_messages for errors
+- Introspect on the `ActiveRecord::Errors` collection object on an AR instance
+  - use `#valid?()`
+  - use `#errors()`
+- Generate `full_message`s for errors
 - Check an attribute for validation errors
-- common validation macros
-  - scope on uniqueness
-  - validation configuration options
 - Add a custom validation error to an AR model
 
 
-## Notes
+# Context: Databases and Data Validity
+
+## AR Validations Are Not Database Validations
+
+Many relational databases such as PostgreSQL have data validation features that
```

Comment:
Students coming to the course probably aren't 100% clear on *what* a validation is. We should define a validation before using it. Also I'd use Sqlite as the example db instead of postgres. That's what we use up until now